lightpath/tests/test_controller.py

```diff
@@ -184,6 +185,6 @@ def test_sim_ctrl():
 
     lc = LightController(sim_client)
 
-    assert len(lc.beamlines.keys()) == 9
+    assert len(lc.beamlines.keys()) == len(beamlines)
```

Comment:
ew did I do this?

Reply:
It's not clear to me if you wrote this or just got credited for moving it from one test file to another.
Either way, it's a reminder to all of us to try harder to understand and review the unit tests.